examples/with-one-login-many-subdomains/frontend/src/App.js

```diff
@@ -43,6 +50,8 @@ SuperTokens.init({
                     // redirect users to their associated subdomain e.g abc.example.com for user abc
                     const claimValue = await Session.getClaimValue({ claim: AllowedDomainsClaim });
                     return "http://" + claimValue[0] + ":3000";
+                } else if (context.action === "RESET_PASSWORD") {
+                    return "/auth/reset-password";
                 }
             },
```

Comment:
what happens if tenant3 (who does not have email password), visits the reset password page:
- after getLoginMethods has finished loading
- before getLoginMethods has finished loading